merch-dropper/src/components/FAQ.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {ExpansionPanel, ExpansionPanelSummary, ExpansionPanelDetails, Typography} from '@material-ui/core';
+
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '70%',
+    height: 500,
+    margin: "auto",
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(18),
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+  panel: {
+    borderBottom: "1px solid F3F3F3"
+  },
+  
+
+}));
+
+const FAQ = () =>{
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+        <h1 style={{textAlign: "center", marginBottom: 50}}>Frequently Asked Questions</h1>
+      <ExpansionPanel className={classes.expansion}>
+      <ExpansionPanelSummary
+        className={classes.panel}
+          expandIcon={<ExpandMoreIcon />}>
+          <Typography className={classes.heading}>What is Stripe?</Typography>
```

Comment:
I would add a section for "What even is Merch Dropper?"
- It's the fastest way to set up a hassle free online storefront with your own designs! Upload designs and create products to create a drop-shipping online shop. Your store is yours, build your brand, raise money, and more.

Reply:
Added